velox/exec/PartitionedOutputBufferManager.h

```diff
@@ -21,12 +21,29 @@
 
 namespace facebook::velox::exec {
 
-// nullptr in pages indicates that there is no more data.
-// sequence is the same as specified in BufferManager::getData call. The caller
-// is expected to advance sequence by the number of entries in groups and call
-// BufferManager::acknowledge.
-using DataAvailableCallback = std::function<
-    void(std::vector<std::unique_ptr<folly::IOBuf>> pages, int64_t sequence)>;
+enum GetDataStatus { SUCCESS, ERR_BUFFER_NOT_FOUND };
```

Comment:
Use enum class and kXxx naming for the values. See https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md